src/cryptography/hazmat/primitives/serialization/pkcs12.py

```diff
@@ -18,6 +18,114 @@
 ]
 
 
+class PKCS12Certificate:
+    def __init__(
+        self,
+        cert: x509.Certificate,
+        friendly_name: typing.Optional[bytes],
+    ):
+        if not isinstance(cert, x509.Certificate):
+            raise TypeError("Expecting x509.Certificate object")
+        if friendly_name is not None and not isinstance(friendly_name, bytes):
+            raise TypeError("friendly_name must be bytes")
```

Comment:
bytes or None

Reply:
Implemented in 41b2d6f63e860c5598c73543e50148317cf60df1.